packages/nodes-base/nodes/Aws/AwsLambda.node.ts

```diff
@@ -186,7 +177,7 @@ export class AwsLambda implements INodeType {
 					errorMessage += `\n\nStack trace:\n${responseData.stackTrace}`;
 				}
 
-				throw new Error(errorMessage);
+				throw new NodeOperationError(this.getNode(), errorMessage);
```

Comment:
needs to be an API error here
```suggestion
				throw new NodeApiError(this.getNode(), responseData);
```